src/dotnet/APIView/APIViewWeb/Models/UsageSampleFileModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace APIViewWeb
+{
+    public class UsageSampleFileModel
+    {
+        public string UsageSampleFileId { get; set; } = IdHelper.GenerateId();
+
```

Comment:
1. Can we have field to store original uploaded file name as well to be used in the future? We might enhance the UI in the future to show sample files uploaded and this will require more user friendly name( which is by default original file name).

2. We also need to have a field to store uploaded user name.

Reply:
Would this value be alright to be null given that the alternative upload method is a text input?
There is an optional title input on both pages however, which I'll assume would override the file name as well. let me know if you would prefer this to not be the case.